snap/snapcraft.yaml

```diff
@@ -337,7 +337,7 @@ parts:
   buildx:
     plugin: nil
     source: https://github.com/docker/buildx.git
-    # Matching `docker buildx version`, installed as a Debian package from Docker's apt repo
+    # Matching `docker buildx version`, installed as a Debian package from Docker's archive repo
```

Comment:
It is [apt repository](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository) or archive. "archive repo" is not right.